angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.ts

```diff
@@ -405,34 +378,30 @@ export class MinesCollectionsAddEditComponent implements OnInit, OnDestroy {
     if (this.isEditing) {
       collection['_id'] = this.collection._id;
 
-      this.factoryService.editCollection(collection).subscribe(async res => {
-        this.recordUtils.parseResForErrors(res);
-
-        // update the records collectionId
-        this.updateRecordsCollectionId(this.collection._id);
-
+      const res = await this.factoryService.editCollection(collection);
+      if (!res || res.modifiedCount !== 1) {
```

Comment:
Should be: 
`if (!res || res.nModified !== 1)` or `if (!res || res.ok !== 1)`

Reply:
3a32d6f8c9e96a7ef201e56a13995a6357f2498e